evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -290,6 +288,32 @@ contract Operator is
     return linkToken.transferAndCall(to, value, data);
   }
 
+  /**
+   * @notice Distribute funds to multiple addresses using ETH send
+   * to this payable function.
+   * @dev Array length must be equal, ETH sent must equal the sum of amounts.
+   * @param receivers list of addresses
+   * @param amounts list of amounts
+   */
+  function distributeFunds(
+    address payable[] calldata receivers,
+    uint[] calldata amounts
+  )
+    external
+    override
+    payable
+    onlyOwner()
```

Comment:
Don't think we need to restrict this to only owner, anyone should be able to call it so that they can top up in an emergency.